tools/package-tools/build-package.ts

```diff
@@ -1,10 +1,12 @@
-import {main as ngc} from '@angular/tsc-wrapped';
 import {readFileSync, writeFileSync} from 'fs';
 import {sync as glob} from 'glob';
 import {join} from 'path';
+import {red} from 'chalk';
+import {PackageBundler} from './build-bundles';
 import {buildConfig} from './build-config';
 import {getSecondaryEntryPointsForPackage} from './secondary-entry-points';
-import {PackageBundler} from './build-bundles';
+
+const {spawn} = require('child_process');
```

Comment:
We should be able to import this with an ES import. e.g `import {spawn} from 'child_process';`